contracts/core/Registry.sol

```diff
@@ -336,6 +334,13 @@ contract Registry is IRegistry, Native, AccessControl, Transfer, Initializable,
     /// ======== Internal Functions ========
     /// ====================================
 
+    /// @dev Internal function used by modifier 'onlyProfileOwner'
```

Comment:
remove this

Reply:
Take a look -> https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3347/
Also this one -> https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3223